packages/sdks-tests/src/e2e-tests/dynamic-unknown-element.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { expect } from '@playwright/test';
+import { test } from '../helpers/index.js';
+
+test.describe('Unknown Element', () => {
+  test.only('unknown element is rendered', async ({ page, packageName }) => {
+    test.skip(packageName === 'react-native');
+    await page.goto('/dynamic-unknown-element');
+
+    const testTagName = await page.locator('test');
+
+    const testTagNameId = await testTagName?.getAttribute('builder-id', { timeout: 10000 });
+    const testTagNameClass = await testTagName?.getAttribute('class', { timeout: 10000 });
```

Comment:
As per https://playwright.dev/docs/api/class-locator#locator-get-attribute

> If you need to assert an element's attribute, prefer [expect(locator).toHaveAttribute()](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-attribute) to avoid flakiness

Also, unless there's a very specific reason, we want to avoid providing manual `timeout` values to any of our tests. Playwright usually takes care of waiting for things to happen. Having to provide these values manually is a sign that something is wrong.